library/src/main/java/com/wultra/android/mtokensdk/operation/IOperationsService.kt

```diff
@@ -103,10 +103,10 @@ interface IOperationsService {
      * If operations are already polling, this call is ignored
      * and the polling interval won't be changed.
      *
-     * @param pollingInterval Polling interval in milliseconds
+     * @param pollingInterval Polling interval in milliseconds, default value is 7s
```

Comment:
Add information that values the minimum is 5s and anything less will be corrected.